datarepo-clienttests/src/main/java/scripts/deploymentscripts/ModularHelmChart.java

```diff
@@ -224,8 +224,6 @@ private void parseAndModifyApiYamlFile(File inputFile, File outputFile) throws I
     // confirm that the expected environment variables/application properties are set
     final List<String> environmentSpecificVariables =
         Arrays.asList(
```

Comment:
```suggestion
        List.of(
```

Reply:
I'm gonna do a separate pass on all of the Arrays.asList things.  I don't want to hold off merging this to unblock the tests